src/App/Pages/Accounts/LoginPasswordlessRequestViewModel.cs

```diff
@@ -243,7 +243,7 @@ private async Task CheckLoginRequestStatus()
                     response = await _authService.GetPasswordlessLoginResquestAsync(_requestId, _requestAccessCode);
                 }
 
-                if (!response?.RequestApproved ?? true)
+                if (!response?.RequestApproved != true)
```

Comment:
⛏️ remove the ! at beginning
```suggestion
                if (response?.RequestApproved != true)
```